codec/address.go

```diff
@@ -4,6 +4,7 @@
 package codec
 
 import (
+	"crypto/rand"
```

Comment:
Can we move this to a separate directory to separate out test code from non-test code?

We should make it `codectest/`

Reply:
Would the following layout work:
```
├── codec
│   ├── address.go
│   ├── address_test.go
│   ├── errors.go
│   ├── hex.go
│   ├── packer.go
│   ├── packer_test.go
│   ├── type_parser.go
│   ├── type_parser_test.go
│   └── utils.go
├── codectest
│   └── codec_test_helpers.go
```
Or should `codectest/` be within `codec`?